Doc/library/codeop.rst

```diff
@@ -43,8 +43,11 @@ To do just the former:
    :exc:`OverflowError` or :exc:`ValueError` if there is an invalid literal.
 
    The *symbol* argument determines whether *source* is compiled as a statement
-   (``'single'``, the default) or as an :term:`expression` (``'eval'``).  Any
-   other value will cause :exc:`ValueError` to  be raised.
+   (``'single'``, the default), as a sequence of statements (``'exec'``) or
+   as an :term:`expression` (``'eval'``).  Any other value will
+   cause :exc:`ValueError` to  be raised. If *symbol* is ``'single'`` and *source*
```

Comment:
The sentence starting with "If symbol is 'single' [...]" can also be dropped. I think these values are already documented at https://docs.python.org/3/library/functions.html#compile so we can keep these docs brief.